.circleci/config.yml

```diff
@@ -151,7 +151,7 @@ jobs:
              --run bash \<<'EOF'
              set -euo pipefail
 
-             DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
I'm not sure I understand how that happens, but apparently `BASH_SOURCE` is not defined on CircleCI.